promotion/claim.go

```diff
@@ -84,6 +90,12 @@ func (service *Service) ClaimPromotionForWallet(
 	if promotion == nil {
 		return nil, errors.New("promotion did not exist")
 	}
+	if !promotion.Active || promotion.ExpiresAt.Before(time.Now()) || time.Since(promotion.CreatedAt) > threeMonths {
```

Comment:
```go
( promotion.CreatedAt < time.Now().AddDate(0,-3,0) ) 
```
Maybe we could use ^^ instead for the 3 month check.

Reply:
that's certainly fine with me. are we thinking this is best for readability?